pom.xml

```diff
@@ -131,6 +131,16 @@
       <id>repo.jenkins-ci.org</id>
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>twitter</id>
+      <url>http://maven.twttr.com</url>
```

Comment:
hrm - not sure about this if it isn't in one of the other repos - not sure we should be adding another one in - doesn't seem right.